squarelet/organizations/tests/factories.py

```diff
@@ -15,6 +15,7 @@ class OrganizationFactory(factory.django.DjangoModelFactory):
     customer = factory.RelatedFactory(
         "squarelet.organizations.tests.factories.CustomerFactory", "organization"
     )
+    allow_auto_join = factory.LazyAttribute(lambda _: False)
```

Comment:
You don't need to use a Lazy Attribute to set it to a constant, you can just set it:
`allow_auto_join = False`